src/endpoint/s3/s3_rest.js

```diff
@@ -226,22 +226,23 @@ async function authorize_request_policy(req) {
     }
 
     const account = req.object_sdk.requesting_account;
-    const is_system_owner = account.email.unwrap() === system_owner.unwrap();
+    // in NSFS NC we don't use email as account identifier
+    const account_identifier = req.object_sdk.nsfs_config_root ? account.name.unwrap() : account.email.unwrap();
```

Comment:
I suspect that there are more places we will need to do that, for example, what about has_bucket_action_permission()?

Reply:
Suspicion is good, we need to plan this change better.

In your example:
https://github.com/noobaa/noobaa-core/blob/32b799e778db9df2bff1ffeaab628d8442c11426/src/server/common_services/auth_server.js#L673-L698

There are a couple of lines that I don't understand how will they work with current configurations:
`bucket.system.owner.email.unwrap()` - in our NSFS NC bucket we don't have a system reference, but `system_owner` is a string, same with `bucket.owner_account.email.unwrap()` we don't have email as a property for the owner_account it is a string.

Reply:
As discussed, I will remove the last 2 commits - the removal of email in schema and internal functions in noobaa will be a gap. In this PR I will only remove the email as a flag that I receive from the user.